Vireo_VS/VireoCommandLine.vcxproj

```diff
@@ -96,7 +96,7 @@
     <UseDebugLibraries>false</UseDebugLibraries>
     <PlatformToolset>v140</PlatformToolset>
     <WholeProgramOptimization>true</WholeProgramOptimization>
-    <CharacterSet>Unicode</CharacterSet>
+    <CharacterSet>MultiByte</CharacterSet>
```

Comment:
Yes, I have this concern too.   Does this refer to the character set of the vcproj or of source code?   GitHub expects/uses UTF-8 encoding for source, correct, and only does client conversion for line endings?